### DISCUSSION TITLE
Pricing

### DISCUSSION THREAD

**Discussion by Zerebokep** [hasura/graphql-engine#9723] [Question]
I'm currently a little bit out of the loop regarding the last pricing update and the shitstorm it caused. Was there any update if the Hasura Team will change the pricing to something realistic or will they keep the current unfair pricing model?

I need to give some answers on this to my boss(es) soon, the project my team is working on will probably need 5000-10000 concurrent websockets (1 per active user) at the launch, probably more when the marketing campaigns start, but with the current pricing (100 websocket cap for $1xxx per month) we'll have to migrate away, as I don't think the enterprise plan is way cheaper than the professional one. :/

**Comment by Zerebokep** [comment]
bump

**Comment by adas98012** [comment]
@Zerebokep 

Our most recent pricing plan is listed [here](https://hasura.io/docs/latest/hasura-cloud/plans/#billing-calculations). Please take a look and evaluate what is best for your use case.
